diff --git a/CMakeLists.txt b/CMakeLists.txt
index 94fab74..c585908 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -34,16 +34,6 @@ if(WITH_BITTORRENT)
 		message(FATAL_ERROR "No Rasterbar libtorrent")
 	endif(LIBTORRENTRB_FOUND)
 	
-	set(asio_DIR ${CMAKE_MODULE_PATH})
-	find_package(asio REQUIRED)
-	
-	if(ASIO_FOUND)
-		message(STATUS "asio found OK ${ASIO_INCLUDE_DIRS}")
-		include_directories(${ASIO_INCLUDE_DIRS})
-	else(ASIO_FOUND)
-		message(FATAL_ERROR "No Boost asio")
-	endif(ASIO_FOUND)
-	
 	set(QT_USE_QTWEBKIT TRUE)
 endif(WITH_BITTORRENT)
 
